{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--9b0f4a01-7e17-462e-9f3a-f9fd5b5d0117",
    "created": "2024-08-13T16:04:41.039795Z",
    "modified": "2024-08-13T16:04:41.0398Z",
    "name": "No Title Available",
    "description": "Incorrect interaction of the parse_packet() and parse_part_sign_sha256() functions in network.c in collectd 5.7.1 and earlier allows remote attackers to cause a denial of service (infinite loop) of a collectd instance (configured with \"SecurityLevel None\" and with empty \"AuthFile\" options) via a crafted UDP packet.",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2017-7401"
        }
    ],
    "references": [
        {
            "name": "97321",
            "tags": [
                "vdb-entry",
                "x_refsource_BID"
            ],
            "url": "http://www.securityfocus.com/bid/97321"
        },
        {
            "name": "RHSA-2017:1285",
            "tags": [
                "vendor-advisory",
                "x_refsource_REDHAT"
            ],
            "url": "https://access.redhat.com/errata/RHSA-2017:1285"
        },
        {
            "tags": [
                "x_refsource_CONFIRM"
            ],
            "url": "https://github.com/collectd/collectd/issues/2174"
        },
        {
            "name": "RHSA-2018:2615",
            "tags": [
                "vendor-advisory",
                "x_refsource_REDHAT"
            ],
            "url": "https://access.redhat.com/errata/RHSA-2018:2615"
        },
        {
            "name": "RHSA-2017:1787",
            "tags": [
                "vendor-advisory",
                "x_refsource_REDHAT"
            ],
            "url": "https://access.redhat.com/errata/RHSA-2017:1787"
        }
    ]
}